rfcs/rfc-email-templates.md

```diff
@@ -0,0 +1,75 @@
+- Start Date: 2021-11-04
+- Target Major Version: [Accommodating email templates](https://github.com/bcgov/entity/issues/9073)
+- Reference Issues:
+- Entity Issue:
+- Implementation PR:
+
+# Summary
+
+Research email templates that support multiple email clients between desktop and mobile devices.
+
+# Basic example
+
+[A sample of responsive email template](http://htmlpreview.github.io/?https://github.com/TedGoas/Cerberus/blob/master/cerberus-hybrid.html)
+
+# Motivation
+
+Nowadays, more and more people use mobile phones to check emails. We must make sure that the email looks good and displays all the details on any device and email client.
+
+The reasons why we had issues of existing email templates:
+
+- Media query
+
+  Media query is a CSS technique introduced in CSS3 for all standard devices. Some email clients like Yahoo or the client is using IMAP protocol are not supported.
+
+- CSS standard
+
+  Outlook for windows only support a subset of the CSS2.1 specification. Also, the gmail clients support different version CSS standards according to [this article](https://www.hteumeuleu.com/2016/trying-to-make-sense-of-gmail-css-support-after-the-2016-update/)
+
+The email clients market share from Litmus Email Analytics.
+
+![Market Share](rfc-email-templates/litmus-email-clients.png)
```

Comment:
This link didn't work for me.  Maybe I don't have access?  No worries, I found the info on the litmus website.

Reply:
https://github.com/bcgov/entity/blob/65b230d6ccbee26c7c9837c61fa71fc4e61d32e4/rfcs/rfc-email-templates.md